### ISSUE TITLE
Callout box coloring not rendering correctly

### ISSUE THREAD

**Issue by wwfiserv** [Fiserv/Support#732]
### Product

Commerce Hub

### Page

https://developer.fiserv.com/product/CommerceHub/docs/?path=docs/test.md&branch=preview

### Description

Developer Studio supported the following themes: Info, Caution, Danger, Success, and Example. Currently the Caution (Orange) and Example (Grey) boxes are not rendering correctly and instead using the default CSS styling of Info (Blue).

Info:

- Provides information that adds clarity to the content, used more as an FYI or note
- Used for a GET endpoint example

Caution:

- Identifies more crucial information that can impact development or an API request
- Used for a PUT endpoint example

Danger:

- Identifies critical information to the developer that severely impacts development or the API
- Used to identify content that is no available yet
- Used for a DELETE endpoint example

Example:

- Used to highlight example text when it can't be displayed as a code block

Success:

- Used for a POST endpoint example

### Expected Behaviour

Example should render in grey and Caution should render in orange

### Operating system

Windows

### Browser

Chrome, Edge

**Comment by minh-pham1** [comment]
Hello! Thanks for bringing up this issue. I'll check with our project manager and see if we have an engineer with bandwidth for this issue, else I'll raise a ticket for a future sprint.

**Comment by minh-pham1** [comment]
Hi William! We support the following:
- `Caution` is actually `Warning` so please feel free to use `<!-- theme: warning -->`
- We don't have `Example`, please simply use no theme tag for the default blue for our standard coloring

If you believe that we should add the `Example` gray box as a supported markdown.

**Comment by wwfiserv** [comment]
@minh-pham1 we actually did have all the versions I mentioned, caution and warning server two different purposes, one is a pay more attention to this it's very important (orange) and the other is hey if you don't do this or do this wrong bad things will happen (red). Similarly we need the separate colors when calling out endpoints like POST, GET, DELETE, PUT to be inline with Swagger and OpenAPI coloring. Also we did have example as a gray box, not sure when this was removed. I only noticed both of these were missing when writing the CH style guidelines, they may have accidentally been removed as part of the ticket I opened about the dark theme colors not being consistent with Microsoft Style Guidelines. As this is a simple CSS styling I wouldn't imagine it would take much to get them added back.

- Info:
  - Provides information that adds clarity to the content, used more as an FYI or note
  - Used for a GET endpoint example
- Caution:
  - Identifies more crucial information that can impact development or an API request
  - Used for a PUT endpoint example
- Danger:
  - Identifies critical information to the developer that severely impacts development or the API
  - Used to identify content that is no available yet
  - Used for a DELETE endpoint example
- Example:
  - Used to highlight example text when it can't be displayed as a code block
- Success:
  - Used for a POST endpoint example

**Comment by minh-pham1** [comment]
Do you still want us to add the Example gray box in our backlog? Our designers may have left it out or opted to use the blue one for accessibility reasons but I can create a backlog enhancement ticket to double check with them if needed.

**Comment by wwfiserv** [comment]
@minh-pham1 yes I would still like one for examples, gray is the default for most sites if a theme is not presented, I know we did light blue. For accessibility it could be another color as well if needed, I thought it used to be light grey. Looking online it seems that notes are usually blue and tips are normally green (which we called success for our endpoints).

Example in Visual Studio Code
![image](https://github.com/user-attachments/assets/eab3944a-d0a2-4163-b57e-820e37094bcc)

**Comment by minh-pham1** [comment]
Ok I'll create a ticket for our design team to research and hopefully they'll come back with the gray colors soon.
